ads/search.py

```diff
@@ -437,8 +438,11 @@ def __init__(self, query_dict=None, q=None, fq=None, fl=DEFAULT_FIELDS,
 
             # Format and add kwarg (key, value) pairs to q
             if kwargs:
-                _ = [u'{}:"{}"'.format(k, v) for k, v in six.iteritems(kwargs)]
-                self._query['q'] = u'{} {}'.format(self._query['q'], ' '.join(_))
+                for field, value in six.iteritems(kwargs):
+                    if not re.match('\s*\(.*\)\s*', value):
+                        # Wrap value in quotes if not already in parentheses
+                        value = u'"{}"'.format(value)
+                    self._query['q'] += u' {}:{}'.format(field, value)
```

Comment:
is the leading space here intentional?

Reply:
If you mean the leading space in `u' {}:{}'` then yes - spaces separate query arguments, no?

Reply:
Yep, that's what I meant. I'd have to deep-dive again to confirm, so thanks for confirming for me!